### PR TITLE
join dissable button

### PR DESCRIPTION
**Pull request by AdityaRaj2097**
### Issue

Properly Disable Join Link #688

Provide a small description in points of what did you change :   We aren't currently accepting new join applications. The link to join RDS is currently disabled by using pointer-events: none, which can be easily overridden from browser devtools and then click on the link. The joining page link is also available in devtools inspect html, which can also be clicked to go to join form. so for solving this we will replace the value of href.

<h1>WHAT IS THE CHANGE :</h1>  removing the value of href from join button give below for dissable join button 

##### Example :

- Done this
- Made these changes

### Is Development Tested? NO

- [ ] Yes
- [x] No

### Before / After Change Screenshots

For visual or interaction changes. Please provide a video / screenshot.

##### Example:

After change image :+1: 

<h3>Test Stats:</h3> 
<p> Beforre</p>

<p> After</p>

### CODE REVIEW

**Comment by ajoykumardas12** [comment]
The PR description is not for complete, please edit. 
PR has to be dev tested and need to add test stats here.

File: index.html [hunk 1]
```diff
@@ -263,7 +263,7 @@ <h1 class="features-title">How can you join?</h1>
           </p>
 
           <div class="wrapper">
-            <a class="btn-join" href="https://realdevsquad.com/join">
+              <a class="btn-join" href="javascript:void(0)">
```
Comment (shubhamsigdar1) [inline index.html:266]: Why it inside div not button??
button has disabled attribute

let me know if I written something wrong?

Comment (AdityaRaj2097) [reply index.html:266]: Yes your are right there should be button but already written code is handling it by anchor tag
Only requirment was that I need to replace value of href

Comment (shubhamsigdar1) [reply index.html:266]: Okk @AdityaRaj2097, let @ajoykumardas12 approve your pr
then i will also approve.

Comment (AdityaRaj2097) [reply index.html:266]: Sure I m working on @ajoykumardas12 changes

Comment (AdityaRaj2097) [reply index.html:266]: @ajoykumardas12 can you tell me how u did attached image here when I m uploading it's showing my image link

**Comment by AdityaRaj2097** [comment]
> The PR description is not for complete, please edit. PR has to be dev tested and need to add test stats here.

@ajoykumardas12 has done the changes kindly check

**Comment by ajoykumardas12** [comment]
There is a lint issue. Please run prettier fix before committing and push new changes.

**Comment by AdityaRaj2097** [comment]
> There is a lint issue. Please run prettier fix before committing and push new changes.

done

**Comment by ajoykumardas12** [comment]
@shubhamsigdar1 Is generate-snapshots supposed to fail here? 
If yes then rest looks good except test stats.

@AdityaRaj2097 why test stats are removed?

**Review by shubhamsigdar1 — APPROVED** [review]

**Comment by AdityaRaj2097** [comment]
> @shubhamsigdar1 Is generate-snapshots supposed to fail here? If yes then rest looks good except test stats.
> 
> @AdityaRaj2097 why test stats are removed?

done , test status , please let me know what is issue why its not working

**Review by ajoykumardas12 — APPROVED** [review]

**Pull request merged**